torchtext/csrc/regex_tokenizer.cpp

```diff
@@ -3,8 +3,8 @@
 
 namespace torchtext {
 
-RegexTokenizer::RegexTokenizer(const std::vector<std::string> &patterns,
```

Comment:
nit: why not use const reference for functions?

Reply:
Which functions?

Reply:
For example, the arguments of the Regex::Sub func are not passed by reference ([link](https://github.com/pytorch/text/pull/1001/files#diff-bbb231ee24be9ba47a93c8b6332cca7cR9)).